### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by wizard-31**
Documentation Added:

Readme
Contributing
Code of Conduct
Code functionality
Tasks
Added relevant badges

### CODE REVIEW

**Review by sak1997 — APPROVED** [review]
Seems fine - good to go!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Contributing to GIT Simplified (GITS)
+# Contributing to Ideal ReposItory for Software projects (I.R.I.S)
```
Comment (sak007) [inline CONTRIBUTING.md]: Rename it to GITS2.0 - Ideal ReposItory for Software projects (I.R.I.S) or something similar

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Please note we have a code of conduct (CODE_OF_CONDUCT.md), please follow it in
 
 ## Pull Request Process
 
-1. Maintain GITS project quality.
+1. Maintain I.R.I.S project quality.
```
Comment (sak007) [inline CONTRIBUTING.md]: Rename

File: README.md [hunk 3]
```diff
@@ -1,162 +1,82 @@
...
-Along with quantitative measures described above, few qualitative measures can help to assess the performance better.
-1. Familiarity with traditional git
-2. hardness of the task
+Here are some measures that can help compare the results between traditional Git/Github Repo Creation and I.R.I.S
+1. The default files created as part of the repo creation.
+2. The structure followed for the default files.
+3. The default template followed for most files, so that even the most layman developer can ensure that his repo adheres to the Standard.
+
+# Team Members
+This repository is made for CSC 510 Software Engineering Course at NC State University.
```
Comment (sak007) [inline README.md]: Add Fall 2021

File: CONTRIBUTING.md (hunk 1 above)
Comment (wizard-31) [reply CONTRIBUTING.md]: Good catch. Made the changes, thanks!

File: CONTRIBUTING.md (hunk 2 above)
Comment (wizard-31) [reply CONTRIBUTING.md]: Good catch. Made the changes, thanks!

File: README.md (hunk 3 above)
Comment (wizard-31) [reply README.md]: Good catch. Made the changes, thanks!

**Review by sak007 — APPROVED** [review]

**Pull request merged**